app/controllers/catalog_controller.rb

```diff
@@ -106,10 +106,13 @@ class CatalogController < ApplicationController
     #  (useful when user clicks "more" on a large facet and wants to navigate alphabetically across a large set of results)
     # :index_range can be an array or range of prefixes that will be used to create the navigation (note: It is case sensitive when searching values)
 
-    config.add_facet_field 'year_facet', label: 'Year', sort: 'index'
-    config.add_facet_field 'group_facet', label: 'Group'
-    config.add_facet_field 'event_type_facet', label: 'Event Type'
-    config.add_facet_field 'game_system_facet', label: 'Game System'
+    config.add_facet_field 'year_facet', label: 'Year', sort: 'index',  solr_params: { 'facet.mincount' => 1 }
```

Comment:
@nomadicoder I'm pretty sure you can set `facet.mincount` in the `solrconfig.xml`.  Though int the interest of putting this out maybe we can leave that as an exercise for another day.